Automattic/studio
# Build DMG files with `appdmg`

Related to https://github.com/Automattic/studio/pull/135.

## Proposed Changes

- Use `appdmg` library to create dmg files.

## Testing Instructions



- Observe CI jobs succeed.
- Observe `Build for Mac` job generates dmg files.
- Download and run dmg files to confirm they work as expected.

## Pre-merge Checklist
